mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -397,6 +401,8 @@ private void tearDownState(Bundle state) {
     public void onDestroyView() {
         super.onDestroyView();
 
+        isAlertsFragmentAvailableForAlerts = false;
```

Comment:
Please remove `ForAlerts` from name. even though we are using them in Alerts option but it is not bound that. So rename booleans in both class to more generic `isAlertsFragmentAvailable` and isTriggersFragmentAvailable